.golangci.yml

```diff
@@ -268,6 +268,10 @@ issues:
     - staticcheck
     text: "SA1019: in.(.+) is deprecated"
     path: .*(api|types)\/.*\/conversion.*\.go$
+  - linters:
+    - staticcheck
+    text: "SA1019: sets.String is deprecated"
```

Comment:
@sbueringer should we open an issue to fix the usage of this deprecated type

Reply:
Yup it's a part of https://github.com/kubernetes-sigs/cluster-api/issues/7925